prow/cmd/deck/static/prow/prow.ts

```diff
@@ -498,20 +508,21 @@ function redraw(fz: FuzzySearch): void {
             if (build.type === "periodic") {
                 r.appendChild(cell.text(""));
             } else {
-                let repo_link = build.refs.repo_link;
-                if (!repo_link) {
-                    repo_link = "https://github.com/" + build.refs.repo;
+                let repoLink = build.refs.repo_link;
+                if (!repoLink) {
+                    repoLink = "https://github.com/" + build.refs.repo;
                 }
                 r.appendChild(cell.link(build.refs.org + "/" + build.refs.repo,
-                    repo_link));
+                    repoLink));
             }
             if (build.type === "presubmit") {
-                r.appendChild(cell.prRevision(build.refs.repo, build.refs.pulls[0]));
+                r.appendChild(cell.prRevision(build.refs.repo || "",
+                    (build.refs.pulls && build.refs.pulls.length > 0) ? build.refs.pulls[0] : Object()));
```

Comment:
Two things:

- Prefer `{}` to `Object()`
- This isn't actually going to do anything useful: `prRevision` implicitly assumes that it's given a valid `Pull` and will emit garbage when given an empty one.

Reply:
Good call, we should just write an empty cell instead (however, that shouldn't happen because presubmits should really have a pull attached).